apis/duck/README.md

```diff
@@ -1,8 +1,9 @@
 # Duck Types
 
 Knative leverages duck-typing to interact with resources inside of Kubernetes
-without explicit knowlage of the full resource shape. `knative/pkg` defines two
-duck types that are used throughout Knative: `Addressable` and `Source`.
+without explicit knowlage of the full resource shape. `knative/pkg` defines
```

Comment:
nit: while here, would you mind:
s/knowlage/knowledge/

Reply:
Well-spotted!